presto-docs/src/main/sphinx/spill/overview.rst

```diff
@@ -44,6 +44,21 @@ memory intensive queries. It is still possible that the query runner will fail
 to divide intermediate data into chunks small enough that every chunk fits into
 memory, leading to `Out of memory` errors while loading the data from disk.
 
+Disk usage
+----------
+
+Spilling intermediate results to disk and retrieving it back is expensive
```

Comment:
In the `admin` docs we describe the properties focusing on recommendation instead of description. IMO the goal of this chapter is to give more intuition of how things work so admins can understand tuning a bit better.